tests/integration/main.go

```diff
@@ -2,40 +2,44 @@ package main
 
 import (
 	"bufio"
+	"flag"
 	"fmt"
 	"log"
 	"os/exec"
 	"reflect"
+	"sync"
 	"time"
 
 	"github.com/kylelemons/godebug/pretty"
 )
 
 var verbose = false
+var localSdkPath = flag.String("local-sdk-path", "",
+	"path of a local sdk version to build and reference in integration tests")
 
 func main() {
-	fmt.Println("============================================ start happy path tests ============================================")
+	flag.Parse()
```

Comment:
I tried running a local sdk path myself and it did not work with this branch. Prob has to do with moving the flag parsing commands around? 

Try 
```bash
go run ./tests/integration/... --local-sdk-path "/Users/bob/Documents/cosmos-sdk/"
```

Reply:
Ah thanks for checking this

Reply:
Fixed it. I was passing a string as param but referencing a global var in the body of the func.